webapp/cube/views.py

```diff
@@ -133,6 +144,149 @@ def cube_microcerts():
     )
 
 
+@login_required
+def get_microcerts():
+
+    # backend_true requires query string,
+    # therefore we need context
+    # also this avoids loading contracts api where not required
+    is_test_backend = (
+        current_app.config["CONTRACTS_TEST_API_URL"]
+        if flask.request.args.get("test_backend", "false")
+        else current_app.config["CONTRACTS_LIVE_API_URL"]
+    )
+    sso_user = user_info(flask.session)
+    if not sso_user:
```

Comment:
Since this view is under a @login_required decorator it should be guaranteed that you have an `sso_user`

Reply:
Also kinda linked to this, if this is a JSON view, it should not be redirecting or returning HTML

Reply:
Good point. Done